plugins/inventory/spacewalk_api.py

```diff
@@ -0,0 +1,183 @@
+#!/usr/bin/env python
+# (c) 2019 Jacob Salmela <me@jacobsalmela.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.0',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = """
+short_description: A dynamic inventory script that uses the Spacewalk API
```

Comment:
```suggestion
short_description: A dynamic inventory plugin that uses the Spacewalk API
```
It's called inventory **plugin**. Inventory scripts are something different.